.codeclimate.yml

```diff
@@ -1,6 +1,11 @@
 engines:
   rubocop:
     enabled: true
+    exclude_fingerprints:
+    # Ignoring long method length for Issue#to_json
+    - 3d618821b1ce28599d6c54f90bb4df59
 ratings:
   paths:
   - "**.rb"
+exclude_paths:
+  - bin/bundler_audit.rb
```

Comment:
Never mind. That doesn't seem to be the issue. I get the false positive locally but not on dotcom. Strange.

```
❯ codeclimate analyze -e rubocop                                         codeclimate-bundler-audit/git/devon/use-bundler-audit-scanner-for-analysis
Starting analysis
Running rubocop: Done!

== bin/bundler-audit (1 issue) ==
1: Use snake_case for source file names. [rubocop]

Analysis complete! Found 1 issue.
```